src/sdk/sql_cluster_router.h

```diff
@@ -260,27 +264,21 @@ class SQLClusterRouter : public SQLRouter {
 
     ::openmldb::base::Status StopJob(int id, ::openmldb::taskmanager::JobInfo& job_info) override;
 
-    ::openmldb::base::Status ExecuteOfflineQuery(const std::string& sql,
-                                                 const std::map<std::string, std::string>& config,
-                                                 const std::string& default_db, bool sync_job,
-                                                 ::openmldb::taskmanager::JobInfo& job_info) override;  // NOLINT
-
-    ::openmldb::base::Status ExecuteOfflineQueryGetOutput(const std::string& sql,
-                                                          const std::map<std::string, std::string>& config,
-                                                          const std::string& default_db,
-                                                          std::string& output);  // NOLINT
+    std::shared_ptr<hybridse::sdk::ResultSet> ExecuteOfflineQuery(const std::string& db, const std::string& sql,
+                                                                  bool is_sync_job, int job_timeout,
+                                                                  ::hybridse::sdk::Status* status) override;
 
     ::openmldb::base::Status ImportOnlineData(const std::string& sql, const std::map<std::string, std::string>& config,
-                                              const std::string& default_db, bool sync_job,
-                                              ::openmldb::taskmanager::JobInfo& job_info) override;
+                                              const std::string& default_db, bool sync_job, int job_timeout,
+                                              ::openmldb::taskmanager::JobInfo& job_info);
```

Comment:
resolve `cpplint` error

Reply:
convert all `::openmldb::taskmanager::JobInfo& job_info` to pointer